### PR TITLE
refactor: bank events and tips events

### PR DESCRIPTION
**Pull request by farrellh1**
# Description

refactor bank events and tips events

Fixes # (issue)

## Contribution

For your PR please add a commnent to eache file edited to explain the changes you made.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

Make sur you run the following commands before submitting your PR:

- /app

```bash
npm run build
npm run test:unit
npm run type-check
npm run lint
npm run format
```

- /backend

```bash
npm run build
npm run test
npm run lint
npm run format
```

- /contract

```bash
npm run test
npm run lint
npm run format
```

- **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by dasarathg68 — APPROVED** [review]
LGTM, fix tests and code quality check

File: app/vitest.config.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import viteConfig from './vite.config'
 import dotenv from 'dotenv'
 
 dotenv.config()
-
+process.env.TZ = 'UTC'
```
Comment (farrellh1) [inline app/vitest.config.ts:9]: is this okay  @hermannleboss ? i need to do this because there is time different in CI and in the local testing

**Comment by farrellh1** [comment]
can you review again @dasarathg68 ?

File: app/src/services/__tests__/bankService.spec.ts
Comment (dasarathg68) [inline app/src/services/__tests__/bankService.spec.ts:1]: Can be removed right?

File: app/src/services/bankService.ts
Comment (dasarathg68) [inline app/src/services/bankService.ts:1]: This file can be removed as well.

File: app/src/services/contractService.ts
Comment (dasarathg68) [inline app/src/services/contractService.ts:1]: This as well.

**Review by dasarathg68 — APPROVED** [review]
LGTM, great work 🚀

**Review by hermannleboss — COMMENTED** [review]
LGTM

**Pull request merged**